.travis.yml

```diff
@@ -44,8 +42,9 @@ jobs:
       script:
         - yarn tx
         - yarn build:$COZY_APP_SLUG:browser
-        - ./scripts/deploy-tests.sh
-        - test TRAVIS_PULL_REQUEST && yarn testcafe:$COZY_APP_SLUG
+        - ./scripts/choose-test-instance.sh
+        - source test.sh
```

Comment:
I don't think this line is supposed to be here?

Reply:
Actually, I need to source test.sh to get my var, as travis doesn't seems to retain var from script : https://github.com/travis-ci/travis-ci/issues/7472#issuecomment-291363061 and https://stackoverflow.com/questions/34021558/travis-ci-using-environment-variables-in-the-deploy-section-of-the-travis-yml